packages/ic-management/src/types/ic-management.params.ts

```diff
@@ -55,7 +58,9 @@ export const toInstallMode = (installMode: InstallMode): InstallModeParam => {
     case InstallMode.Reinstall:
       return { reinstall: null };
     case InstallMode.Upgrade:
-      return { upgrade: null };
+      // TODO: Support Upgrade mode skipping pre-upgrade
+      // `upgrade` can also have `[{ skip_pre_upgrade: [] | [boolean] }]`
+      return { upgrade: [] };
```

Comment:
You double checked that`upgrade: []` is indeed the backwards compatible value that execute and upgrade with pre-upgrade hook? 

Just asking to be sure that there is no hidden reason that would make the default different and require the param to actually be `{ upgrade: [{ skip_pre_upgrade: [false]]}` for backwards compatibility.

Reply:
No, I didn't test the functionality of the method. I don't have any project to test that. I expected the canister to maintain same functionality on the backwards-compatible flow.

I'll ask the execution team.

Reply:
According execution team it's correct `upgrade: []`